js/effect/blur.js

```diff
@@ -1,30 +1,51 @@
 var blurParameters = {
     'scale': [3, 1, 7]
 };
+var bgParameters = {
+    'scale': [3, 1, 7]
+};
 let curBlurS = blurParameters['scale'][0];
 
-function enableBlurEffect() {
+//seperate background blur from canvas
+let BlurS = bgParameters['scale'][0];
+
+function enableBlurEffect(key) {
```

Comment:
I will suggest to create a completely new file for background blur instead of using the same blur.js file.
This will keep our `effect` files for organized for each effect having their own files.